src/MailpitCommands.ts

```diff
@@ -291,12 +266,7 @@ class MailpitCommands {
 				url: this.mailpitUrl(`/view/${messageId}.html`, false),
 				auth: this.auth,
 			})
-			.then((response) => {
-				if (response.status !== 200) {
-					throw new Error(`Failed to get message body. Status: ${response.status}, Body: ${response.body}`);
-				}
-				return response.body as string;
-			});
+			.its('body');
```

Comment:
can we not change the return here ?

Reply:
.its('body') is returning the body which is casted to MessageType but if it is wanted I can change it.

https://docs.cypress.io/api/commands/its > Get a property's value on the previously yielded subject.

```suggestion
			.then(response => response.body as string);
```

I think both working the same.